app/controllers/transcriptions_controller.rb

```diff
@@ -110,21 +147,35 @@ def whitelisted_attributes?
     update_attrs.keys.all? { |key| update_attr_whitelist.include? key }
   end
 
-  def approving?
-    update_attrs["status"] == "approved"
+  def update_attr_whitelist
+    ["flagged", "text", "status"]
   end
 
   def allowed_filters
     [:id, :workflow_id, :group_id, :flagged, :status, :internal_id]
   end
 
-  def update_attr_whitelist
-    ["flagged", "text", "status"]
+  def approving?
+    update_attrs["status"] == "approved"
   end
 
   def jsonapi_serializer_params
     {
       serialize_text: action_name == 'show'
     }
   end
+
+  def if_unmodified_since
+    since = request.headers['If-Unmodified-Since']
+
+    if since.blank?
+      raise ValidationError, 'Missing header "If-Unmodified-Since", action cannot be performed.'
```

Comment:
I think this wording needs to clarify what is expected to be supplied and how to do it.
```suggestion
      raise ValidationError, "You must supply the 'If-Unmodified-Since' header and it must contain the resource's GET request 'Last-Modified' header value'
```

Reply:
Yes... agreed, additional clarification in this error name will be helpful for API users.